docs/metrics.md

```diff
@@ -56,12 +60,14 @@ An event fired when the user actively navigates away from the Notes sidebar. Inc
 - `cd7`
 - `cd8`
 - `cd10`
+- `cd11`
 
 #### `changed`
-An event fired when the user completes a change of the content of the notepad. It prospectively begins when a user focuses on the notepad's editable area, and ends when the user either 1) closes the sidebar, or 2) does not make any changes in 20 seconds. Includes:
+An event fired when the user completes a change of the content of the active notepad. It prospectively begins when a user focuses on the notepad's editable area, and ends when the user either 1) closes the sidebar, or 2) does not make any changes in 20 seconds. Includes:
 
 - `ec` - `notes`
 - `ea` - `changed`
+- `ev` - `<integer index of active note>`
```

Comment:
"do people work on top notes", need to see what to do about `index`, because we sort.